src/minimega/vnc_cli.go

```diff
@@ -103,36 +137,37 @@ func cliVNC(c *minicli.Command, resp *minicli.Response) error {
 		return err
 	} else if c.BoolArgs["playback"] {
 		// Start keyboard playback
-		return vncPlaybackKB(host, vm, fname)
+		return vncPlaybackKB(vm, fname)
 	}
+	return nil
+}
 
-	// List all active recordings and playbacks
-	resp.Header = []string{"host", "name", "id", "type", "filename"}
-	resp.Tabular = [][]string{}
+func cliVNCClear(c *minicli.Command, resp *minicli.Response) error {
```

Comment:
Should we do something akin to `cliVNCClear` when a VM is flushed? What cleans up these maps?